MFVNeutralino/test/TrackMover/Samples.py

```diff
@@ -0,0 +1,1528 @@
+#!/usr/bin/env python
```

Comment:
Mentioned elsewhere, but let's get rid of the multiple Samples.py and SampleFiles.py. It will only cause us trouble.

Reply:
Remove it. Done.